spec/services/halls/create_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Halls::Create do
+  let(:params) do
+    {
+      name: 'name',
+      capacity: 100
+    }
+  end
+  let(:instance) { described_class.new(params) }
```

Comment:
```suggestion
  subject { described_class.new(params).call }
```

In the other test cases you use subject, please stick to one convention.